app/src/main/java/com/peacecorps/pcsa/MainActivity.java

```diff
@@ -46,7 +46,11 @@
 import java.util.LinkedList;
 import java.util.List;
 
-
+/**
+ * This class shows the first aide section (home screen) of the application.
+ * @author chizoba
```

Comment:
The name here should be the author of the class which is Rohands.
We can always track that you added the comment via git log